components/currency-buttons.tsx

```diff
@@ -1,23 +1,26 @@
 "use client";
 
+import { useRouter, useSearchParams } from "next/navigation";
 import { Button } from "@/components/ui/button";
-import { usePathname, useRouter, useSearchParams } from "next/navigation";
 import { cn } from "@/lib/utils";
 import { DEFAULT_DISPLAY_CURRENCY } from "@/configs/hypercerts";
 
-export const CurrencyButtons = ({ className }: { className?: string }) => {
-  const pathname = usePathname();
-  const searchParams = useSearchParams();
+interface CurrencyButtonsProps {
+  value?: string;
+  className?: string;
+}
+
+export const CurrencyButtons = ({ value, className }: CurrencyButtonsProps) => {
```

Comment:
Value prop isn't being passed on user profile when viewing marketplace orders so button doesn't show correct state.

Reply:
Good catch, thanks! Fixed in #311